### ISSUE TITLE
Add ability to inject SQL before built-in statements

### ISSUE THREAD

**Issue by moltar** [berenddeboer/cdk-rds-sql#6]
My request is rather a niche one, so I can see how this may not be desired.

I need an "escape hatch" and run some SQL _before_ the role is created.

https://github.com/berenddeboer/cdk-rds-sql/blob/2ee7dd1e969bb7b0d7291faa0d6b66f49c34498b/src/handler.ts#L86

This is related to the following issue, where in QuickSight the new password encryption is not supported and needs to be downgraded:

https://stackoverflow.com/a/74527856/1566758

Basically, I need to run this, before role creation:

```sql
set password_encryption = 'md5';
```

**Comment by berenddeboer** [comment]
Happy to see a PR for this with this as an option.

**Comment by berenddeboer** [comment]
@claude can you let me know if QuickSight still does not support the new password encryption?
